tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -503,3 +503,8 @@ Then('the clipboard content should match permalink of resource {string}', functi
 Then('the thumbnail should be visible in the app-sidebar', function () {
   return client.page.FilesPageElement.appSideBar().isThumbnailVisible()
 })
+
+When('the user uploads overwriting file {string} using the webUI and retries if the file is locked', function (file) {
```

Comment:
there is no "retry" when the file is locked. what happens when a file is locked in phoenix?